src/igvfd/schemas/whole_organism.json

```diff
@@ -0,0 +1,73 @@
+{
+    "title": "Whole Organism",
+    "$id": "/profiles/whole_organism.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submiting a whole organism sample.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "sample.json#/properties"
+        },
+        {
+            "$ref": "biosample.json#/properties"
+        },
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/accession"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/documents"
+        },
+        {
+            "$ref": "mixins.json#/product_info"
+        },
+        {
+            "$ref": "mixins.json#/url"
+        },
+        {
+            "$ref": "mixins.json#/references"
+        },
+        {
+            "$ref": "mixins.json#/taxa"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "biosample.json#/dependentSchemas"
+        ]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "sex": {
+            "title": "Sex",
+            "description": "Sex of the whole organism sample.",
+            "type": "string",
+            "default": "unspecified",
+            "enum": [
+                "male",
+                "female",
+                "unspecified"
+            ]
```

Comment:
I think this is a calculated property

Reply:
i think should be removed form the schema

Reply:
I saw that and I changed my test, but I forgot to remove it from here:( Done!